lib/hubstats/github_api.rb

```diff
@@ -87,6 +87,35 @@ def self.update_pulls
       end
     end
 
+    # Public - Gets information on teams, e.g. name
+    #
+    # Returns - nothing
+    def self.update_teams
+      grab_size = 250
+      begin
+        client = Hubstats::GithubAPI.client
+        incomplete = client.organization_teams("sportngin")
```

Comment:
Needs to use the org name from the configuration, not hard coded to ours.
Also, this variable name could be better.